pom.xml

```diff
@@ -38,16 +38,15 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.1.10.RELEASE</version> <!-- see also https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-parent -->
-		<!-- Warning! Cannot bump to 2.1.11.RELEASE due to missing adoption by spring-cloud.version Greenwich.SR4 -->
+		<version>2.2.2.RELEASE</version> <!-- see also https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-parent -->
```

Comment:
The version of awaitility is reported to be superfluous now (see also https://github.com/promregator/promregator/pull/144/files#diff-600376dffeb79835ede4a0b285078036R201 )